Tests/CMakeLists.txt

```diff
@@ -97,20 +97,18 @@ if (OGRE_BUILD_TESTS)
         set(CROSS ${CROSS} -DCMAKE_CXX_FLAGS=-stdlib=libc++)
     endif()
     
-    set(GTEST_INSTALL_DIR ${CMAKE_BINARY_DIR}/gtest)
```

Comment:
what is the issue you are trying to solve here? the tests build (and run) fine as is on the CI:
https://ci.appveyor.com/project/paroj/ogre-bsrh7/branch/master